src/webapp/components/DeliveryTable.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import { makeStyles } from "@material-ui/core/styles";
+import { useTranslation } from "react-i18next";
+
+const useStyles = makeStyles({
+  container: {
+    maxHeight: 440,
+  },
+});
+
+const DeliveryTable = ({ rows }) => {
+  const { t: str } = useTranslation();
+  const classes = useStyles();
+
+  // sort happens in-place
+  rows.sort((rowA, rowB) => rowB.timestamp - rowA.timestamp);
+
+  // format data for presentation
+  const formattedRows = rows.map((row) => {
+    const timestamp = new Date(row.timestamp * 1000);
+
+    return {
+      ...row,
+      timestamp: `${timestamp.toLocaleDateString()}: ${timestamp.toLocaleTimeString()}`,
+    };
+  });
+
+  return (
+    <TableContainer className={classes.container} component={Paper}>
+      <Table aria-label="simple table">
+        <TableHead>
+          <TableRow>
+            <TableCell>{str("webapp:zoneFinder.label.code")}</TableCell>
+            <TableCell align="right">
+              {str("webapp:zoneFinder.label.crossStreetFirst")}
+            </TableCell>
+            <TableCell align="right">
+              {str("webapp:zoneFinder.label.crossStreetSecond")}
+            </TableCell>
+            <TableCell align="right">
+              {str("webapp:zoneFinder.label.firstName")}
+            </TableCell>
+            <TableCell align="right">
+              {str("webapp:zoneFinder.label.slackLink")}
+            </TableCell>
+            <TableCell align="right">
+              {str("webapp:zoneFinder.label.timestamp")}
+            </TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {formattedRows.map((row) => (
+            <TableRow key={row.Code}>
+              <TableCell component="th" scope="row">
+                {row.Code}
```

Comment:
can you replace these row field strings with those in lib/airtable/tables/requests?

Reply:
@mjmaurer sure! But how do I import _~airtable/tables/requests_ into the react app?

If I follow, we're using these row field strings to avoid weird property names ("Cross Street #1").

`import Requests from "~airtable/tables/requests";`  yields

> Module not found: Can't resolve '~airtable/tables/requests' in '/Users/wmattgardner/mutual-aid-app/src/webapp/components'

Importing with relative path throws errors about the dependencies of the thing being imported.

Any ideas? Am I totally missing something? Thanks!

Reply:
hey @alllthesignals! Michael is out atm, Sherminn here. I think we probably don't have path aliasing set up for the react app, so you might just have to use relative path e.g. `../../lib/airtable`. Since this is non-blocking, plus we have other places in the webapp where we need to fix this, im going to go ahead and merge this first and follow up with a new issue to handle hardcoding of airtable field keys. 

This table view will be important for the upcoming work for https://github.com/crownheightsaid/mutual-aid-app/issues/122, where volunteers can pick up delivery straight from the map.  thank you for your contribution!